src/headers/SiteHeader.scss

```diff
@@ -4,15 +4,16 @@
   --background-color: var(--bloom-color-white);
   --base-align-items: inherit;
   --base-desktop-padding: inherit;
-  --base-desktop-width: inherit;
+  --base-desktop-width: 1025px;
```

Comment:
Just removed an unused class name that was overriding this value, with exactly this value